lib/util.js

```diff
@@ -60,40 +58,26 @@ export function privateKeyDerEncode({privateKeyBytes, seedBytes}) {
     // extract the first 32 bytes of the 64 byte private key representation
     p = Buffer.from(privateKeyBytes.buffer, privateKeyBytes.byteOffset, 32);
   }
-  const keyBuffer = new ByteBuffer(p);
-
-  const asn1Key = asn1.create(
-    asn1.UNIVERSAL,
-    asn1.Type.OCTETSTRING,
-    false,
-    keyBuffer.getBytes()
-  );
-
-  const a = asn1.create(
-    asn1.Class.UNIVERSAL,
-    asn1.Type.SEQUENCE,
-    true, [
-      asn1.create(
-        asn1.Class.UNIVERSAL, asn1.Type.INTEGER, false,
-        asn1.integerToDer(0).getBytes()),
-      // privateKeyAlgorithm
-      asn1.create(asn1.Class.UNIVERSAL, asn1.Type.SEQUENCE, true, [
-        asn1.create(
-          asn1.Class.UNIVERSAL,
-          asn1.Type.OID,
-          false,
-          asn1.oidToDer(oids.EdDSA25519).getBytes()
-        ),
-      ]),
-      // private key
-      asn1.create(
-        asn1.Class.UNIVERSAL, asn1.Type.OCTETSTRING, false,
-        asn1.toDer(asn1Key).getBytes()),
-    ]
-  );
-
-  const privateKeyDer = asn1.toDer(a);
-  return Buffer.from(privateKeyDer.getBytes(), 'binary');
+  // we can use the same DER for each key because keyLength is always 32 bytes
+  const DERBuffer = Buffer.from([
+    0x30,
+    0x2e,
+    0x02,
+    0x01,
+    0x00,
+    0x30,
+    0x05,
+    0x06,
+    0x03,
+    0x2b,
+    0x65,
+    0x70,
+    0x04,
+    0x22,
+    0x04,
+    0x20
+  ]);
```

Comment:
Should make this is a constant `DER_PRIVATE_KEY_PREFIX` at the top of the file and it can be a single line for example, it could be:

```js
DER_PRIVATE_KEY_PREFIX = Buffer.from('<hex>', 'hex');
```
And can break like this if needed:

```js
DER_PRIVATE_KEY_PREFIX = Buffer.from(
  '<hex>', 'hex');
```

Reply:
Addressed here: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/5/commits/83c242815aeff1e173bc9c1ca4bf6bdb4f38ad50